### PR TITLE
Ft routes

### PR DESCRIPTION
**Pull request by Rwandarushya**
#### What does this PR do?
Can CREATE Route
Can GET All Routes
Can GET Route
Can UPDATE Route
Can DELETE Route
#### Description of Task to be completed?
Can CREATE Route
Can GET All Routes
Can GET Route
Can UPDATE Route
Can DELETE Route
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
N/A
#### What are the relevant Trello boards stories?
https://trello.com/c/ZG69cG4N
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: src/controller/routeController.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import model from '../database/models';
+
+// eslint-disable-next-line import/prefer-default-export
```
Comment (malaba6) [inline src/controller/routeController.js]: Do not disable eslint rules. You should rather conform to them

File: src/controller/routeController.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    orginID: req.body.origin,
+    destinationID: req.body.destination,
+    busStops: req.body.busStops
+  };
+
+  model.Route.create(rt)
+    .then((data) => {
+      res.status(201).json({ status: 201, message: 'Route created successfully', data });
+    })
+    .catch((err) => res.status(500).json({ status: 404, message: err }));
```
Comment (malaba6) [inline src/controller/routeController.js]: Why do u return here two different status codes?

File: src/database/models/index.js [hunk 3]
```diff
@@ -1,37 +1,33 @@
...
 } else {
   sequelize = new Sequelize(config.database, config.username, config.password, config);
 }
-
 fs
   .readdirSync(__dirname)
   .filter((file) => (file.indexOf('.') !== 0) && (file !== basename) && (file.slice(-3) === '.js'))
   .forEach((file) => {
     // eslint-disable-next-line global-require
+    // eslint-disable-next-line import/no-dynamic-require
```
Comment (malaba6) [inline src/database/models/index.js]: same here

**Review by malaba6 — CHANGES_REQUESTED** [review]

File: src/controller/routeController.js (hunk 2 above)
Comment (Rwandarushya) [reply src/controller/routeController.js]: I removed the second ones

File: src/controller/routeController.js (hunk 1 above)
Comment (Rwandarushya) [reply src/controller/routeController.js]: Now it is corrected

File: src/controller/routeController.js (hunk 2 above)
Comment (fistonhn) [reply src/controller/routeController.js]: error
Marius can you bring back capital T in this file? other than that you did a great job as yesterday we had a meeting for reviewing and resolving errors.

**Review by Imaraika — APPROVED** [review]

**Review by Imaraika — COMMENTED** [review]
I have clone your branch and run it locally  it is working.

**Pull request merged**